manifests/postgresql.crd.yaml

```diff
@@ -162,6 +208,22 @@ spec:
                     type: string
             replicaLoadBalancer:  # deprecated
               type: boolean
+            connectionPool:
```

Comment:
This part is duplicate, except for the enum part under `mode`. Can you copy the latter to the position above?
I would like to have alphabetical sorting in the validation because it's so huge already and I think things are easier to find later.